spec/acceptance/server/grant_role_spec.rb

```diff
@@ -65,60 +65,6 @@ class { 'postgresql::server': }
     MANIFEST
   end
   let(:pp_two) do
-    <<-MANIFEST.unindent
```

Comment:
is this no longer used ?

Reply:
It's identical to the first manifest. So I decided to just use it and run the check's of the same apply.

Reply:
Used a comparison checker to make sure.